### ISSUE TITLE
Training results

### ISSUE THREAD

**Issue by vladmandic** [vladmandic/onetrain#8]
using fine-tuned base model and batch-size:2
![Image](https://github.com/user-attachments/assets/00cc5967-5a5b-45ea-a257-6dbd82f9bdd0)

**Comment by vladmandic** [comment]
using fine-tuned base model and batch-size:1
![Image](https://github.com/user-attachments/assets/94f2aaae-e778-4f10-b3e4-f158dd92b066)

**Comment by vladmandic** [comment]
using fine-tuned base model and adafactor optimizer
![Image](https://github.com/user-attachments/assets/fb5dcf04-7738-4000-abcd-a4e422b248af)

**Comment by vladmandic** [comment]
using original base model and low warmup:
![Image](https://github.com/user-attachments/assets/81202097-d979-4148-8e89-aac7ef87feb0)

**Comment by vladmandic** [comment]
using original base model and high warmup for more precise learning rate:
![Image](https://github.com/user-attachments/assets/106a7d44-f4a8-42b8-8b24-6491faecc5ba)
